packages/communication-react/review/beta/communication-react.api.md

```diff
@@ -77,6 +79,12 @@ import { VideoDeviceInfo } from '@azure/communication-calling';
 import { VideoStreamRenderer } from '@azure/communication-calling';
 import { VideoStreamRendererView } from '@azure/communication-calling';
 
+// @public (undocumented)
+export type ACSCallManagement = {
```

Comment:
Is there a reason to define seperate types for ACSCallManagement and TeamsCallManagement or can we use a generic?
```
interface CallManagement<CallType> {
    joinCall(microphoneOn?: boolean): <CallType> | undefined;
    startCall(participants: string[], options?: StartCallOptions): <CallType> | undefined;
};
```